blocks/cards/mockData.json

```diff
@@ -0,0 +1,6008 @@
+{
+  "total": 14237,
+  "offset": 0,
+  "limit": 500,
+  "data": [
+    {
+      "date": "45126",
+      "image": "cards-placeholder/media_1d995a89dc32e47de6829f5a205848e0d9fc374ce.jpeg?width=2000&format=webply&optimize=medium",
```

Comment:
@DriftingSands the image value should start with `/` as you can see from the rest of the data.